md/wildfly-bundle.md

```diff
@@ -164,19 +170,38 @@ WildFly can be shut down by executing the following script:
 
 You can also press Ctrl + C.
 
+<a id="post-install" />
```

Comment:
According to https://github.com/bonitasoft/bonita-doc/pull/190/commits/91aa7934df91e601dba83c00b1088d181f3f2bf3, you did it (maybe not on purpose).
I'll remove it.